frontend/src/components/PromptSpeech.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import Speech from "react-speech";
+
+const PromptSpeech = (props) => {
```

Comment:
i think it would be better of doing ({ prompt }) here instead of (props) cuz later you can just do text={prompt} in your speech component